lib/ecstatic/showdir.js

```diff
@@ -129,12 +130,15 @@ module.exports = function (opts, stat) {
             }
 
             var displayName = he.encode(file[0]) + ((isDir)? '/':'');
+            var ifile = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAN1wAADdcBQiibeAAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAAAC0SURBVDiN7Y8tjsMwGESn1kgBAQEGoSELs7fojdpb9FShYd4gh4TmCt+PXVReV4U70sD3NHNZ1/VqZo9Syi8aEkJIJG+XZVnSNE3zOI4IIbwFl1JwnieO4/ijiMwxRohIywDEGJFznunucPcmGABeHM0MtdZmAQCYGaiqHwtUFXT3jwXfufAv+JJAVUGyGTYzBJI5pQQRQa31rYoIUkogmdn3/X3f98e2bT8tC7qu24dhuD8BD6e7SzzK9MwAAAAASUVORK5CYII=";
```

Comment:
@jfhbrook you just have to encode the image in [Base64](https://en.wikipedia.org/wiki/Base64).
Here's the code I am using:
```python
with open(path_to_image, 'rb') as f:
    print f.read().encode('base64').replace('\n', '')

```